test/testUtils.ts

```diff
@@ -178,6 +178,37 @@ function triggerFakeMouseEvent(type: string, target: D3.Selection, relativeX: nu
   target.node().dispatchEvent(e);
 }
 
+function triggerFakeTouchEvent(type: string, target: D3.Selection, relativeX: number, relativeY: number) {
```

Comment:
NBD but should the argument to trigger here be the selection?  It seems specific enough to the DOM element that the node should be passed in not the selection.